app/src/utils/DatasetTypes.ts

```diff
@@ -25,6 +25,22 @@ export interface Field {
 // TODO: make typedef for valid data types instead of any.
 export type Row = Readonly<Record<string, any>>;
 
+export interface Variable {
+  readonly variableId: string;
+  readonly variableName: string;
+  readonly description: string;
+  readonly datasetIds: readonly string[];
+  readonly rows: readonly Row[];
+}
+
+export interface Breakdowns {
+  geography: "national" | "state" | "county";
+  // Note: this assumes only one demographic breakdown at a time. If we want to
+  // support more later we can refactor this to multiple boolean fields.
+  demographic?: "race" | "age" | "gender";
+  date?: boolean;
```

Comment:
when would we use "date"? would this be more likely to be a date range?

Reply:
Yeah I was thinking this would be for whether or not a dataset is grouped by date or not. For example, requesting covid data with `date: true` would give you covid data by date, whereas if you just want total covid counts without a date you'd pass `date: false` (or leave it unset). I think specifying date ranges falls into a separate category of date filtering, which I'm not sure if we'll need, but we can always add later. The one thing I think could be worth doing here is specifying this as a grouping window. Eg, you could specify `time: "day"` to have one row per day, or `time: "week"` to have one row per week. But anyway, I didn't want to complicate this until we know what the use case will be - I think it should be pretty easy to change when we start working with time range data